.github/workflows/backend-cd-workflow.yml

```
@@ -36,43 +36,42 @@ jobs:
       - name: Upload build artifact
         uses: actions/upload-artifact@v3
         with:
-          working-directory: ./backend
           name: friendogly-be-develop-jar
-          path: ./**/friendogly-0.0.1-SNAPSHOT.jar
+          path: ./backend/build/libs/friendogly-0.0.1-SNAPSHOT.jar
           
   deploy:
     needs: build
     runs-on: self-hosted
     steps:
-      - name: change permission
+      - name: Change permission
         run: |
           sudo chown -R ubuntu:ubuntu /home/ubuntu/actions-runner/_work/2024-friendogly 
+
       - name: Download build artifact
         uses: actions/download-artifact@v3
         with:
           name: friendogly-be-develop-jar
           path: ./backend/build/libs
 
-      - name: Check if room-esc server is running on port 8080
-        id: check-server-on-port
+      - name: Find Java Application PID And Kill PID
+        id: find_pid
+        run: |
+          sudo pgrep -f friendogly-0.0.1-SNAPSHOT.jar | xargs sudo kill -9 | true
+
+      - name: Check if room-esc server is running on port 8080 after stop
```

```suggestion
      - name: Check if friendogly server is running on port 8080 after stop
```
별건 아니지만... room-esc.. ㅋㅋㅋ